File: app/src/main/java/com/artmurka/artmurkaapp/Presenter/CategorySettingsPresenter.java
package com.artmurka.artmurkaapp.Presenter;

import com.artmurka.artmurkaapp.Presenter.InterfacesPresenter.ICategorySettingsPresenter;
import com.artmurka.artmurkaapp.Views.Activities.IMainActivity;
import com.artmurka.artmurkaapp.Views.Activities.MainActivity;
import com.artmurka.artmurkaapp.Views.Fragments.Interfaces.ICategorySettings;

import java.util.HashMap;

public class CategorySettingsPresenter implements ICategorySettingsPresenter {
    ICategorySettings fragment;
    public CategorySettingsPresenter(ICategorySettings fragment){
        this.fragment = fragment;
    }


    @Override
    public void applyChanges(HashMap<String, String> settings, int sort) {



    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/presenter/CategorySettingsPresenter.kt
package com.artmurka.artmurkaapp.presenter

import com.artmurka.artmurkaapp.presenter.interfaces_presenter.ICategorySettingsPresenter
import com.artmurka.artmurkaapp.android.views.fragments.interfaces.ICategorySettings
import java.util.HashMap
import javax.inject.Inject

class CategorySettingsPresenter @Inject constructor(): BasePresenter<ICategorySettings>(),ICategorySettingsPresenter {


    override fun applyChanges(settings: HashMap<String, String>, sort: Int) {

    }
}
